packages/components/psammead-radio-schedule/src/ProgramCard/__snapshots__/index.test.jsx.snap

```diff
@@ -51,6 +51,15 @@ exports[`ProgramCard should render correctly for live 1`] = `
   margin: 0;
 }
 
+.c4 {
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  color: #B80000;
+  display: inline-block;
+  margin-right: 0.5rem;
```

Comment:
The live label component has this much margin at all times. It looks a little much to me when in use here, in the bulletins, and in normal story promos, but I think it would be good to use the live label component in here now and then take another look - we can change it everywhere in one place then :D